src/go/service/router.go

```diff
@@ -23,23 +24,28 @@ type Router interface {
 	SctpdDownlinkClient() sctpd.SctpdDownlinkClient
 	// SctpdUplinkClient provides a sctpd.SctpdUplinkClient
 	SctpdUplinkClient() sctpd.SctpdUplinkClient
+	// ConfigClient provides a config.ConfigClient
+	ConfigClient() config.ConfigClient
```

Comment:
Hmm... who would use this?

Extreme view: we could have all internal readers of Config use this; thus Config would become a service?

Reply:
ill remove it was looking at it more as maybe the internal tests are running on multiple boxes and need to talk to each other but ill leave it out for now and bring it back if we need it later on. 
reading configs internally over grpc can lead to a ton of problems i think we should avoid.